CHANGES.md

```diff
@@ -30,7 +30,10 @@ breaking changes, and mappings for the large list of deprecated functions.
 
 ### Changes between 3.0.3 and 3.0.4 [xx XXX xxxx]
 
- * none yet
+ * Case insensitive string comparison no longer uses locales.  It has instead
```

Comment:
s/locales/locale objects/ ?

Reply:
I think it's okay either way but there is no mentioned of locale objects in the earlier changes entry.